src/unit_tests/analysisd/test_exec.c

```diff
@@ -14,12 +14,14 @@
 #include <stdio.h>
 
 #include "../wrappers/wazuh/os_net/os_net_wrappers.h"
+#include "../wrappers/wazuh/os_regex/os_regex_wrappers.h"
 #include "../wrappers/wazuh/shared/labels_op_wrappers.h"
 #include "../wrappers/wazuh/wazuh_db/wdb_global_helpers_wrappers.h"
 #include "../wrappers/wazuh/shared/mq_op_wrappers.h"
+#include "../wrappers/wazuh/shared/validate_op_wrappers.h"
 #include "../../analysisd/eventinfo.h"
 #include "../../analysisd/config.h"
-#include "../../analysisd/alerts/exec.h"
+#include "../../analysisd/alerts/exec.c"
```

Comment:
Why did you changed this?

Reply:
I changed this because the prototype of the function tested is in exec.c